### DISCUSSION TITLE
Page build and development keep queued

### DISCUSSION THREAD

**Discussion by ilovemath25** [community/community#116886] [New to GitHub]
Hi! I'm pretty new to Github and i have some problem arround here.
When I'm trying to deploy website in github, the action keep having queued status
![image](https://github.com/community/community/assets/147607832/f030ca7c-b0df-466f-b73d-a456d77ac801)
anyone know why is this happened?

**Comment by Linuus** [comment]
https://www.githubstatus.com/

**Comment by jfullstackdev** [comment]
your site is now live, if you are still not aware, if you still have questions, you can ask more

![image](https://github.com/community/community/assets/47092464/f376cc8e-8fd5-4b48-9849-a08c6560c967)
